### PR TITLE
LPS-134466 Test Autom Documentation Link

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**JIRA Ticket:**
LPS-134466
LPS-117015

**Motivation:**
Test Automation for "As an instance administrator, I want to change the link to documentation included in the vocabularies help text"

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
@CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase [hunk 1]
```diff
@@ -87,6 +87,84 @@ definition {
...
+			Category.removeLinkToDocumentationURL();
+		}
+
+		task ("Assert if Learn How is not displayed") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContentNavigator.gotoAddCP();
+
+			AssertElementNotPresent(locator1 = "VOCABULARY_LINK_DOCUMENTATION");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase:104]: Don't you need to put the path file before? like "Category#VOCABULARY_LINK_DOCUMENTATION"??

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase [hunk 2]
```diff
@@ -87,6 +87,84 @@ definition {
...
+	test CheckLearnHowLinkNotDisplayedVocabularies {
+		task ("Remove default link to Documentation") {
+			Category.removeLinkToDocumentationURL();
+		}
+
+		task ("Assert if Learn How is not displayed") {
+			Category.openCategoriesAdmin(siteURLKey = "global");
+
+			AssertElementNotPresent(locator1 = "VOCABULARY_LINK_DOCUMENTATION");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase:118]: Don't you need to put the path file before? like "Category#VOCABULARY_LINK_DOCUMENTATION"??

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase [hunk 3]
```diff
@@ -87,6 +87,84 @@ definition {
...
+
+		task ("Assert if Learn How is not displayed") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+
+			AssertElementNotPresent(locator1 = "VOCABULARY_LINK_DOCUMENTATION");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase:134]: Don't you need to put the path file before? like "Category#VOCABULARY_LINK_DOCUMENTATION"??

**Comment by CarlosBrichete** [comment]
@Luizcarlosqueiroz Please review the comments I added. If this is ok, you can forward it

**Comment by Luizcarlosqueiroz** [comment]
ci:test:tango

**Comment by Luizcarlosqueiroz** [comment]
ci:forward

**Pull request closed without merging**